src/plugins/math/pureMath/storeMath.ts

```diff
@@ -0,0 +1,35 @@
+import { AnyAction } from '@reduxjs/toolkit';
+import { legacy_createStore as createStore } from 'redux';
+
+
```

Comment:
look at the conversation of meeting
no need switch with one case
camelCase for functions naming
why the type  named through the constant?